<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB</class>
 <widget class="QWidget" name="DB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 253, 240);</string>
  </property>
  <widget class="QTableWidget" name="tabWid_bd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 234);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>Filtres</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_history">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_csv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>511</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_history_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
